packages/phishing-controller/src/PhishingController.test.ts

```diff
@@ -65,6 +66,134 @@ describe('PhishingController', () => {
     const controller = getPhishingController();
     expect(controller.state.whitelist).toStrictEqual([]);
   });
+  it('should return false if the hostname is in the whitelist', async () => {
+    const whitelistedHostname = 'example.com';
+
+    nock(PHISHING_CONFIG_BASE_URL)
+      .get(METAMASK_STALELIST_FILE)
+      .reply(200, {
+        data: {
+          // eslint-disable-next-line @typescript-eslint/naming-convention
+          eth_phishing_detect_config: {
+            allowlist: [whitelistedHostname],
+            blocklist: [],
+            fuzzylist: [],
+          },
+          // eslint-disable-next-line @typescript-eslint/naming-convention
+          phishfort_hotlist: {
+            blocklist: [],
+          },
+          tolerance: 0,
+          version: 0,
+          lastUpdated: 1,
+        },
+      })
+      .get(`${METAMASK_HOTLIST_DIFF_FILE}/${1}`)
+      .reply(200, { data: [] });
+
+    nock(CLIENT_SIDE_DETECION_BASE_URL)
+      .get(C2_DOMAIN_BLOCKLIST_ENDPOINT)
+      .reply(200, {
+        recentlyAdded: [],
+        recentlyRemoved: [],
+        lastFetchedAt: 1,
+      });
+
+    const controller = getPhishingController();
+    controller.bypass(formatHostnameToUrl(whitelistedHostname));
+    await controller.updateStalelist();
```

Comment:
Why are we doing this and using `nock` for this test? It seems like we're testing if the whitelist works - I'm not sure what methods add to the whitelist beyond bypass but `updateStalelist()` doesn't touch it.